### ISSUE TITLE
[Bug]: Endeavour 1.9.0 needs Forge update to be compatible with Oracle's java

### ISSUE THREAD

**Issue by bwilcox** [FTBTeam/FTB-Modpack-Issues#1201]
### Modpack

Endeavour

### Modpack version

1.8.0

### Describe the bug

Host: Freebsd 13-0-Release-p7
Server was powered off unexpectedly.  On reboot the minecraft server failed to start due to a mod with a corrupt config.  I attempted to reinstall the mods from the installer and I see messages like this:

```
2022/08/01 13:04:09 Getting downloads for Forge Install
Error downloading : Get "": unsupported protocol scheme ""
Error downloading : Get "": unsupported protocol scheme ""
```

And ultimately I see this message:

```
2022/08/01 13:04:24 Downloaded 957 successful, 231 failed, 0 incomplete.
Some downloads failed. Would you like to continue anyway? (y/n) [y]
```

I also attempted a total install in an empty directory and had the exact same problem.

### Steps to reproduce

1. Freebsd 13.0 host.  Openjdk8.
2. Download the Endeavour 1.8 server installer for Freebsd.
3. Run the installer.

### Expected behaviour

I expected all of the mods to download and install.

### Additional information

I did attempt to install the 1.9.0 beta of endeavour in a new directory and that installed as expected.  However, that I have java issues with.  I would prefer sticking with the 1.8.0 release of Endeavour if possible.

**Comment by Raidobw2** [comment]
Hmm, there has been mentions of that problem in the past days. I'm not exactly sure why that happens, since I can't replicate it. I've installed Endeavour 1.8.0 successfully on Windows 10 just now.

Posting the server's crash log would have helped as well, but I think I know what the issue is already.

1. If you open your world's folder, you'll find a folder called `serverconfig`. Delete files inside the serverconfig folder (all of them).
2. In your server or client installation there is a folder on the root of the modpack folder called `defaultconfigs`. Copy everything inside, and paste those files inside `world/serverconfig`
3. Start your server, should be fixed.
4. If any errors persist you can take the `config` folder of your client and put it on the server, that works for the case it's one of the configs there instead

If you want to update to 1.9.0 instead and have a weird Java bug, let me know what that bug is and I might be able to help. Currently we have a command in the FTB Discord (`!olderjava`) which prints this:

> Some modpacks require older versions of Java 8 or 11. Open https://adoptium.net/temurin/archive/ in your browser and select either OpenJDK 8 or OpenJDK 11. You can install those exact versions from the archive:
> MC 1.16.5 or less => `jdk8u312-b07` from 21st of October 2021.
> MC 1.16.5 for performance => `jdk-11.0.13+8` from 21st of October 2021.
> Servers: uninstall all other versions of Java, fix your path variable or directly reference the path to the java binary if you still get an error. Example: https://discord.com/channels/372448486723158016/560188834592063488/934734172419936277.

In short, if you get a Java error on 1.9.0, make sure you're using Java 11 (11.0.13+8) or a lower Java 11 version. This was an issue with Forge//Oracle at some point and I think that's fixed in 1.16.5 by updating the Forge version, but we haven't gotten to it yet possibly for that modpack.

**Comment by bwilcox** [comment]
@Raidobw2 Thank you for the response.  I appreciate the steps on the server crash, but my attempts to reinstall have already made that process impossible to complete.  I had deleted the mods and config directories prior to my attempt to reinstall.  So the problem with 1.8.0 on FreeBSD is that the installer cannot download some of  the mods now.

I did spend some time the other night with the beta and can confirm that updating the Forge version did fix the java issue and my server is running again using the 1.9 beta pack.  So you have my vote for that fix.

Out of curiosity, I also updated the forge version in a fresh but incomplete, due to the error I've reported here,  1.8 install.  It did get past the java error but did error out on missing required mods, which was expected based on the download issue.

If I had a vote, I would say update the Forge version in 1.9 and make it a release rather than spend time on an issue that apparently is only impacting FreeBSD servers at 1.8.  Assuming there aren't other issues in the beta that need to be worked out that is.  I'm still curious what changed, as I've been running a 1.8 server for almost a year now.  So the installation was working at one point.

If there is some kind of debugging from the installer that can be enabled for more information, I'd be happy to provide more logs if we want to pursue the 1.8 installer issue.

**Comment by Raidobw2** [comment]
Glad to see that worked for 1.9.0, I'll convert this request to remind us to update Forge, please leave it opened.

As for the server installer, I did ask the team for clues and apparently it could be due to old server installers. If you haven't downloaded the server installer in a while for 1.8.0, you can try to download it again and run it as a test. If that works, the problem was solved when they updated that binary a few days back, when API changes had been made.

Cheers

**Comment by Newbytee** [comment]
I'm not sure if it's related, but FTB Endeavour won't work on the latest versions of Java due to this issue: https://github.com/McModLauncher/modlauncher/issues/91

**Comment by manmaed** [comment]
This issue is stale
